ldj-spring-renderer-example/src/main/resources/com/infosupport/ldoc/springexample/ampq/template.adoc

```diff
@@ -0,0 +1,21 @@
+= Spring AMPQ Sandwich Example
+:nofooter:
+
+This "sandwich shop" "implementation" uses Spring `AMPQ`, as follows:
+
+[plantuml,"diagram",svg]
+----
+@startuml
+skinparam SequenceMessageAlignment reverseDirection
+skinparam SequenceGroupBodyBackgroundColor Transparent
+skinparam SequenceBoxBackgroundColor #Gainsboro
+skinparam SequenceArrowThickness 2
+skinparam BoxPadding 10
+skinparam ParticipantPadding 10
+skinparam LifelineStrategy solid
+hide footbox
+' diagram
+@enduml
+----
+
+For more information about Spring AMPQ, see https://docs.spring.io/spring-amqp/reference/html/#introduction[the Spring Framework reference].
```

Comment:
Nit: Technically this is the Spring AMQP reference doc, not the Spring Framework reference doc, I think.